### DISCUSSION TITLE
✅ Toolkit v0.3

### DISCUSSION THREAD

**Discussion by flanakin** [microsoft/finops-toolkit#499] [Releases]
### 🆕 New in 0.3
- 📗 [FinOps documentation](https://microsoft.github.io/finops-toolkit/#-learning-resources) and other learning resources have been added.
- 📊 [Power BI reports](https://aka.ms/ftk/pbi) have all been updated and a new Data ingestion report was added to provide insights into storage.
- 📒 [Cost optimization workbook](https://microsoft.github.io/finops-toolkit/optimization-workbook) now respects the global subscription filter.
- 🖥️ [FinOps toolkit PowerShell commands](https://aka.ms/ftk/ps) to backfill Cost Management exports.
- 🌐 [Open data files](https://microsoft.github.io/finops-toolkit/data) for Services now includes ServiceModel and Environment columns.

### ℹ️ Updated
- 🏦 [FinOps hubs](https://aka.ms/finops/hubs) bug fixes and improvements.
- 📊 [Power BI reports](https://aka.ms/ftk/pbi) had many fixes and improvements across all reports!
- 📒 [Governance workbook](https://microsoft.github.io/finops-toolkit/governance-workbook) improvements for manage disks, combined governance and security tabs, and enabled the export button.
- 🖥️ [FinOps toolkit PowerShell commands](https://aka.ms/ftk/ps) fixes to adapt to Cost Management API changes.

📋 Milestone tasks: [Toolkit v0.3](https://github.com/microsoft/finops-toolkit/milestone/18)
🌱 Feature branches: [dev](https://github.com/microsoft/finops-toolkit/tree/dev)

---
### 📥 [Download v0.3](/microsoft/finops-toolkit/releases/v0.3) &nbsp; &nbsp; &nbsp; 📜 [Changelog](https://aka.ms/ftk/changes#-v03) &nbsp; &nbsp; &nbsp; 📗 [Learn more](https://aka.ms/finops/toolkit) &nbsp; &nbsp; &nbsp; ❤️ [Give feedback](https://aka.ms/ftk/feedback)

**Comment by flanakin** [comment]
Update: Renamed this release to 0.2.1.

**Reply by flanakin** [reply]
We decided to bump this to 0.3 given all the things that went into it.

**Reply by saleemmy** [reply]
https://microsoft.github.io/finops-toolkit/hubs
In this link it says 0.2

**Reply by flanakin** [reply]
The references there are for when we updated to 0.2. I can see how that's confusing. I'll get those updated for the next release. Thanks for calling it out!

**Comment by majknowak** [comment]
Hi All :)
 
Firstly let me congratulate on all of the work you all do here! This is amazing!
 
I am a part of BI Team that was recently asked by our Client to implement FOCUS to our existing cloud cost reporting. 
We've began the discovery of all the available docs etc., and right now are a bit overwhelmed tbh.
 
Could you please share some brief guide on the available options and use cases?
 
Our Client is a international organization with approx. $3M main cloud providers spend per month.
Current architecture includes PBI Premium workspace and different connectors (Cost Management, KQL via Log Analytics API, Athena ODBC (AWS CUR), Big Query connector (GCP Billing tables).
 
I've tested the CM PBI Report as a sort of PoC and the feedback was very good. We want to go deeper into FOCUS, but now miss a bit of an idea what to do next?
We are experimenting with Biq Query Focus view and converters.
 
Please be so kind and show us the way :)
BR,
Michał

**Reply by flanakin** [reply]
Check out https://microsoft.github.io/finops-toolkit/focus to get started with FOCUS. Let us know if there's anything specific you'd like to see added to this or if there's anything confusing. We also offer some [reports that will help you get started with FOCUS](https://aka.ms/ftk/pbi), whether you're using the Cost Management connector or [FinOps hubs](https://aka.ms/finops/hubs).

If you have a Microsoft account rep, you can also speak to them. Feel free to mention my name and I can help guide them down the right path.

**Reply by majknowak** [reply]
Hi Michael,
Thanks for your reply! I know all of the stuff that's already there and what I am missing is a sort of roadmap of steps to implement.
Especially in terms of multicloud customer.
I am a bit blind on what would I need to do, to have a report looking like the you've shared for CM but including my costs of AWS, GCP and Azure all together.

Can you help me understand it better?

**Reply by flanakin** [reply]
It's possible to ingest GCP data. @thecloudman implemented this on top of hubs in his environment. We're discussing contributing that back into hubs, which should help a lot.

As for AWS, the challenge is getting it converted to FOCUS. You could use the FinOps Foundation's FOCUS converter. Once that happens, you should be able to just drop the files in the `ingestion` container in hubs and you're good. We aren't at a place where we've been able to fully test that flow, but that's the ultimate goal.
